tests/test_engine_responses_tool_calls.py

```
@@ -52,11 +144,12 @@ async def get_response(self, *_args, **_kwargs) -> str:
             return ""
 
     api_client = _Client()
+
+    def _has_pending_tool_call(client):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_has_pending_tool_call`

(ANN202)